source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,310 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page you will learn the how to search **geospatial data** in the
+MongoDB Java Driver.
+
+Geospatial data is data that represents a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with **GeoJSON**.
+GeoJSON is a geospatial data format created by the Internet Engineering
+Task Force (IETF). You can use the GeoJSON format to store your data in MongoDB,
+and you must use the GeoJSON format to query your geospatial data in MongoDB. 
+
+For a concrete reference, GeoJSON data often resembles the following structure:
+
+.. code-block:: json
+
+   "<field name>" : {
+      "type": "<GeoJSON type>",
+      "coordinates": [
+         [<longitude>, <latitude>],
+         ...
+         [<longitude>, <latitude>]
```

Comment:
The prior format showed that the longitude and latitude pairs were different by enumerating them. This might be confusing since it doesn't differentiate whether the placeholders contain the same values. E.g.
<longitude_1>, <lattitude_1>
...
<longitude_n>, <lattitude_n>

Reply:
Thanks for the note. I agree these should be enumerated. This ended up being removed in turning this into the "location of MongoDB headquarters" example here: https://github.com/mongodb/docs-java/pull/78/commits/28d9772c7463b4c18d3522eb444788775b0bde66

Going forward, I will try to enumerate placeholder lists from `1 to n`.